src/libraries/Directory.Build.props

```diff
@@ -94,15 +94,14 @@
     <MonoEnvOptions Condition="'$(MonoEnvOptions)' == '' and '$(TargetsMobile)' != 'true' and '$(MonoForceInterpreter)' == 'true'">--interpreter</MonoEnvOptions>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(TargetsMobile)' == 'true'">
```

Comment:
This condition was there so that we don't set any of these properties unnecessarily when not testing workloads.

Reply:
This was fine earlier when we were building Wasm/Wasi.Build.Tests with targetOS=wasi/browser, but now we are building them for the host os, and thus these properties would need to be available for `TargetsMobile!=true`.

The Was*Build.Tests run on coreclr, and each test then builds and runs test projects for wasm.

Reply:
Is there another feature flag that we can condition on here?

Reply:
hmm not a feature flag really, but we could have a property to indicate that we are building a workload-testing dependent project, and use that here.

Reply:
I think that would be good.